portfolio/src/main/webapp/index.html

```diff
@@ -24,6 +24,18 @@ <h1>Fact method</h1>
       <button onclick="addRandomMyFact()">Fax</button>
       <div id="fact-container"></div>
       <p>Click <a href="aboutMe.html">here</a> to learn more about me!</p>
+      <!-- button to view data servlet-->
```

Comment:
I really like the use of buttons you added, very simple and clear!

Reply:
Thank you!